tests/test_paperqa.py

```diff
@@ -544,6 +558,49 @@ def accum(x):
     await docs.aquery("What is the national flag of Canada?", answer=answer)
 
 
+@pytest.mark.skipif(
+    not (os.environ.get("ANYSCALE_BASE_URL") and os.environ.get("ANYSCALE_API_KEY")),
+    reason="Anyscale URL and keys are not set",
+)
+@pytest.mark.asyncio()
+async def test_anyscale_chain():
+    client = AsyncOpenAI(
+        base_url=os.environ["ANYSCALE_BASE_URL"], api_key=os.environ["ANYSCALE_API_KEY"]
+    )
+    llm = OpenAILLMModel(
+        config={
+            "temperature": 0,
+            "model": "meta-llama/Meta-Llama-3-8B-Instruct",
+            "max_tokens": 56,
```

Comment:
Can you document why 56 with a code comment? Or make this set by a formula?

Reply:
it's arbitrary -- other tests use 56, i can do 50 I suppose.